src/main/java/core/basesyntax/handler/impl/PurchaseOperationHandler.java

```diff
@@ -0,0 +1,16 @@
+package core.basesyntax.handler.impl;
+
+import core.basesyntax.handler.OperationHandler;
+import core.basesyntax.model.FruitTransaction;
+import core.basesyntax.storage.Storage;
+
+public class PurchaseOperationHandler implements OperationHandler {
```

Comment:
Don't begin class or method implementation with an empty line. The class `PurchaseOperationHandler` starts with an empty line which should be removed according to the checklist.